frontend/common/types/responses.ts

```diff
@@ -92,6 +93,12 @@ export type UserPermission = {
 export type GroupPermission = Omit<UserPermission, 'user'> & {
   group: UserGroup
 }
+export type FeatureStateValue = {
+  boolean_value: boolean
```

Comment:
I think this is an incorrect duplicate of FeatureStateValue on line 165

Reply:
Actually if I'm not mistaken, both are wrong and they should be nullable rather than allow for undefined (perhaps best to check on BE)

Reply:
The duplicated type was removed, and the fields were updated by verifying the BE.